### PR TITLE
Updated address definitions

### PR DESCRIPTION
**Pull request by RubenKruiswijk**
By running generate.php

Important change is the format for Ireland with the addition of the iercode.

### CODE REVIEW

File: resources/address_format/CH.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
 		"locality",
 		"postalCode"
 	],
-	"uppercase_fields": [],
```
Comment (bojanz) [inline resources/address_format/CH.json]: This is odd, how did that happen? Difference in json encoding between our systems?

Comment (RubenKruiswijk) [inline resources/address_format/CH.json]: I don't know, could be, will that be a problem? Do you want me to revert that file?

Comment (bojanz) [inline resources/address_format/CH.json]: Yes, please.

Comment (RubenKruiswijk) [inline resources/address_format/CH.json]: Done!

**Comment by bojanz** [comment]
Thank you!

**Pull request merged**